scripts/create_res_drawable.R

```diff
@@ -0,0 +1,471 @@
+## From the Android docs
+## https://developer.android.com/guide/topics/resources/drawable-resource
+## "A drawable resource is a general concept for a graphic that can be drawn to the screen"
+## This script creates the graphics that are shown underneath certain puzzle descriptions
+## For example, app/src/main/res/drawable-hdpi/backgammon_small.png
+## is shown in a puzzle about the probability of certain jumps in the game of backgammon
+
+library(ggplot2)
+library(grid)
```

Comment:
Would be nice to provide a Docker container (or a Dockerfile) with R + these libraries installed